errors/stack_trace.go

```diff
@@ -65,16 +64,15 @@ func (f frame) formatContext() string {
 
 // GetRuntimeContext returns function name and code line.
 func GetRuntimeContext() (fileName, funcName string, line int) {
-	st := *callers()
+	st := callers()
 	frame := frame(st[1])
 	fileName, funcName, line = frame.getContext()
 	return
 }
 
-func callers() *stack {
+func callers() []uintptr {
```

Comment:
This function (and probably a few others) is copied directly from the pkg/errors: https://github.com/pkg/errors/blob/master/stack.go#L163
The x-ray SDK supports both the generic `[]uintptr` as well as the pkg/errors stacktrace format directly, so it seems strange that we need to change the code from pkg/errors to support the x-ray SDK. Any thoughts on this? :thinking:

Reply:
Unaware of this having been copied, I've just removed this with it being redundant (at least a custom type without any methods declared seemed so).

I reworked the solution to implement the `pkg/errors` StackTrace interface instead, since this package depends on it anyway. We could possibly update it to the latest version or to one matching that of aws-xray-sdk-go.